src/style.rs

```diff
@@ -1,6 +1,14 @@
 use super::utils::*;
 use egui::{style::Margin, *};
 
+/// Left or right alignment for tab add button.
+#[derive(Clone, Copy, Debug, PartialEq, Eq, Hash)]
+#[cfg_attr(feature = "serde", derive(serde::Deserialize, serde::Serialize))]
+pub enum TabAddAlign {
```

Comment:
While this type is pub in the `style` module, the module itself is not exposed by the crate, making this type inaccessible from the outside.
I suggest adding it to the `pub use` section [here](https://github.com/Avarel/egui_dock/blob/tab_add/src/lib.rs#L70).